content/hi/docs/home/_index.md

```diff
@@ -0,0 +1,67 @@
+---
+approvers:
```

Comment:
```suggestion

```

Reply:
oh yes!! I saw this & had left that for the last !!
Thanks for pointing it out 👍